packages/dx-react-scheduler-material-ui/src/templates/views/vertical/time-scale/layout.jsx

```diff
@@ -85,6 +108,8 @@ export const Layout = ({
         cellsData={cellsData}
         className={classes.ticks}
         groupOrientation={groupOrientation}
+        groupCount={groupCount}
+        includeAllDayCell={showAllDayTitle}
```

Comment:
For future. We should not use specific names. I mean that this component calls TicksLayout and this component should not have information about grouping)